astronet/predict.py

```diff
@@ -90,15 +93,86 @@ def predict(legacy=False):
 
     results = pd.DataFrame.from_dict(series)
     
-    if FLAGS.output_file:
-      with tf.io.gfile.GFile(FLAGS.output_file, "w") as f:
+    if output_file:
+      with tf.io.gfile.GFile(output_file, "w") as f:
         results.to_csv(f)
         
     return results, config
 
 
+def checkpoint_dirs(root: str, nruns: Optional[int] = None) -> list[str]:
```

Comment:
Could we use your new checkpoint function? This one is a bit archaic with the `os.path` usage and the runtime checks.

Reply:
Pull it out into a util function.